pkg/tracking/service/experiments.go

```diff
@@ -89,6 +90,23 @@ func (ts TrackingService) RestoreExperiment(
 	return &protos.RestoreExperiment_Response{}, nil
 }
 
+func (ts TrackingService) UpdateExperiment(
+	input *protos.UpdateExperiment,
+) (*protos.UpdateExperiment_Response, *contract.Error) {
+	experiment, err := ts.Store.GetExperiment(input.GetExperimentId())
+	if err != nil {
+		return nil, err
+	}
+	if input.NewName != nil {
+		experiment.Name = utils.PtrTo(input.GetNewName())
```

Comment:
Not feeling strongly about this one but would it not make sense to just assign `experiment.Name = input.NewName` I would assume these are both pointers. These are the things where I'm not sure what the proper Go thing to do it.

Reply:
I don't mind the `if` check, but just wonder whether `input.GetNewName()` (getting string value from pointer) only to wrap it again as a pointer is what we want here. It feels like an extra step.